test/order_test.rb

```diff
@@ -1,9 +1,9 @@
-require 'minitest/autorun'
-require 'minitest/reporters'
-require 'minitest/skip_dsl'
+require "minitest/autorun"
+require "minitest/reporters"
+require minitest/skip_dsl"
 
-require_relative '../lib/customer'
-require_relative '../lib/order'
+require_relative ""../lib/customer"
```

Comment:
Syntax error extra `"`.

```suggestion
require_relative "../lib/customer"
```